docs/ROADMAP.md

```diff
@@ -11,7 +11,7 @@ If you have a feature request or suggestion, please [get in touch](https://githu
 ## [v1.0-rc.1 (Nov 2024)](https://github.com/spiceai/spiceai/milestone/45)
 
 - Bugfixes and performance improvements
-- Data accelerators: Arrow, DuckDB, SQLite, PostgreSQL (Beta)
+- Data Accelerators: Arrow & DuckDB  (Release Candidate)
```

Comment:
DuckDB might as well get brought in as connector RC for this release too, if the accelerator will be RC